components/FloatingContact/src/index.scss

```diff
@@ -39,6 +39,13 @@
   z-index: var(--denhaag-floating-contact-index);
 }
 
+@media (min-width: 1024px) {
+  .denhaag-floating-contact {
+    --denhaag-floating-contact-bottom: 2rem;
```

Comment:
I would rather see that the denhaag spacing token is used.

example: `--denhaag-floating-contact-bottom: var(--denhaag-space-block-2xl);`  and `--denhaag-floating-contact-right: var(--denhaag-space-inline-2xl);`

Reply:
Aangepast